usr/share/regolith-look/dracula/compositor-init

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+# See https://regolith-linux.org/docs/customize/compositors/
+
+compositor_PID=$(pidof picom)
+
+while [ ! -z "$compositor_PID" ]; do
+  kill $compositor_PID
+  sleep .1
+  compositor_PID=$(pidof picom)
+done
+
+if [[ -f "$HOME/.config/regolith2/picom/config" ]]; then
+  /usr/bin/picom --config "$HOME/.config/regolith2/picom/config"
+else
+  /usr/bin/picom --config /usr/share/regolith-look/nord/picom.conf
```

Comment:
this path is invalid, it needs to point to your version of the picom.conf.  I think it's just swapping `nord` for `dracula`.

Reply:
Yep, you are correct. I will push a fix shortly.

Reply:
I have fixed the path and re-pushed.